README.md

```diff
@@ -1,47 +1,47 @@
 # Dorm social room manager (frontend)
 
-This is a repository for the frontend part of the dorm social room manager. Made using React + MaterialUI and
-Typescript.
+This is a repository for the frontend part of the dorm social room manager. Made using React + MaterialUI and Typescript.
 
 ## Requirements
 
 Before you will use this repository make sure that you have installed:
 
-- ***Git***
+- **_Git_**
```

Comment:
Why you have changed formatting? Please revert old formatting

Reply:
Must've been an accident. Fixed in the latest commit.

Reply:
this is still not fiex